ape-config.yaml

```diff
@@ -0,0 +1,39 @@
+plugins:
+  - name: solidity
+  - name: ganache
+
+contracts_folder: contracts/contracts
+
```

Comment:
Can we also move the `.build` directory not to be on the toplevel, but under `contracts/`?

Reply:
this is not possible with ape

Reply:
now inside contracts